aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/AWSCognitoAuthPlugin.kt

```diff
@@ -72,6 +79,23 @@ class AWSCognitoAuthPlugin : AuthPlugin<AWSCognitoAuthServiceBehavior>() {
 
     @VisibleForTesting
     internal lateinit var realPlugin: RealAWSCognitoAuthPlugin
+    private val queueFacade: KotlinAuthFacadeInternal by lazy {
+        KotlinAuthFacadeInternal(realPlugin)
+    }
+
+    private val queueChannel = Channel<Job>(capacity = Channel.UNLIMITED).apply {
```

Comment:
Can we add plugin level scope instead of using `GlobalScope`?

`private val pluginScope = CoroutineScope(Job() + Dispatchers.Default)`